ex22.py

```diff
@@ -0,0 +1,20 @@
+file = "namesfile.txt"
```

Comment:
Why parse the data twice? Can you not use the generator or iterator to parse the file & then do the next check?